pkg/install/install.go

```diff
@@ -124,33 +131,38 @@ func NewInstaller(ctx context.Context, log *logrus.Entry, env env.Interface, db
 
 // Install installs an ARO cluster
 func (i *Installer) Install(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
-	steps := map[api.InstallPhase][]func(context.Context) error{
+	steps := map[api.InstallPhase][]struct {
+		action    func(context.Context) error
+		condition wait.ConditionFunc
+		timeout   time.Duration
+	}{
 		api.InstallPhaseDeployStorage: {
-			i.createDNS,
-			func(ctx context.Context) error {
+			{i.createDNS, nil, 0},
```

Comment:
write `{action: i.createDNS}` - the other values are zero values and can be omitted for clarity